README.md

```diff
@@ -52,8 +52,10 @@ https://iree.dev/developers/general/github-actions/.
     bugs, and other work tracking
 *   [IREE Discord server](https://discord.gg/wEWh6Z9nMU): Daily development
     discussions with the core team and collaborators
-*   [iree-discuss email list](https://groups.google.com/forum/#!forum/iree-discuss):
-    Announcements, general and low-priority discussion
+*   [iree-announce email list](https://lists.lfaidata.foundation/g/iree-announce):
+    Announcements
```

Comment:
How about we also put a link to the community calendar in here? https://zoom-lfx.platform.linuxfoundation.org/meetings/iree (unless there's a better link)

Reply:
Absolutely. The calendar (and no, there is no better link) should be linked somewhere. As the upper section is titled *Communication Channels* I hesitated to add it to the list. Let me add a subsection, similar to *Related Project Channels* and let's see if this would work.

Reply:
I don't have a particularly strong opinion either way. Could rename "communication channels" to something more generic like "community", "project channels", or "project resources"

Reply:
Okay, let me revert adding the calendar link as part of this PR and add it in a follow up. It feels like we probably want to restructure a little more (right now the community meetings are linked at https://github.com/iree-org/iree/blob/main/README.md#presentations-and-talks and there probably should be a separate *community meetings* section instead).